### ISSUE TITLE
Option to group same task timers together across days

### ISSUE THREAD

**Issue by zpalmtree** [hamaluik/timecop#14]
If you start a timer for the same issue multiple times, they will be grouped together allowing you to see the total time spent. If the timer spans multiple days however, this is not the case.

It would be nice to have an option to allow us to easily see the total time spent for a long running task.

Thanks for the great app btw!

**Comment by hamaluik** [comment]
I’m not sure the dashboard is the best place to view that information.. I’m having a hard time imagining how it would work there. The largest version (1.5.0) added a table report of time sums by project, would that work for you? Perhaps if I added the ability to filter by searched description in the reports as well?

**Comment by zpalmtree** [comment]
Yep, I imagine it would be tricky to nicely represent in the UI. Adding description filtering would work for me.
